website/server/libs/inbox/index.js

```diff
@@ -80,6 +80,76 @@ export async function getUserInbox (user, options = {
   return messagesObj;
 }
 
+async function usersMapByConversations (owner, users) {
```

Comment:
I think this might be the result of some merge issues because the code for this is currently in website/server/libs/inbox/conversation.methods.js

Reply:
@paglias Yes, I must have made a mistake during merging.  Not only the function usersMapByConversations, but also the listConversations function (which is supposed to be in conversation.methods.js file also) are incorrectly merged into this file in my previous PR.  The two functions have now been deleted from the .../inbox/index.js file in the new push.